### PR TITLE
updated html

### PR DESCRIPTION
**Pull request by tusharkandpal**
- arranged links in order
- corrected profile avatar
- updated iframe links

### CODE REVIEW

File: components/alert/alert.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="shortcut icon" href="../../assets/Wings.svg" />
-    <link rel="stylesheet" href="./alert.css" />
+    <link rel="stylesheet" href="../../utilities/doc.css" />
     <title>Docs | Alert</title>
```
Comment (junaidshaikh-js) [inline components/alert/alert.html]: Instead of docs in the title you can use your library name

File: components/alert/alert.html [hunk 2]
```diff
@@ -27,20 +27,20 @@ <h1 class="home-header-title"><a href="/">WINGS UI</a></h1>
       <aside class="doc-sidebar">
         <ul class="sidebar-list ls-none mp-m">
           <strong class="sidebar-header font-m">Components</strong>
```
Comment (junaidshaikh-js) [inline components/alert/alert.html:29]: `ul` should have `li` as direct child, wrap strong inside `li`

**Review by junaidshaikh-js — COMMENTED** [review]
Good Work!

File: components/alert/alert.html (hunk 1 above)
Comment (tusharkandpal) [reply components/alert/alert.html]: yeah i will change it. Thanks!

File: components/alert/alert.html (hunk 2 above)
Comment (tusharkandpal) [reply components/alert/alert.html:29]: this is like the heading of the list. we can do this atleast i think.

Comment (junaidshaikh-js) [reply components/alert/alert.html:29]: Probably we can move it outside of ul then

Comment (tusharkandpal) [reply components/alert/alert.html:29]: i'm not sure of doing it now. will do after all components are done.

**Review by Namrata-J — APPROVED** [review]
looks good to go with merging!!

**Review by Raghav888 — APPROVED** [review]
Approved

**Pull request merged**